### PR TITLE
[CI] Add labeler

### PR DESCRIPTION
**Pull request by SaschaMann**
There's already a config file for it

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -10,3 +16,116 @@ type/mentor-bios:
 
 type/mentor-notes:
 - tracks/*/exercises/*/mentoring.md
+
+# Tracks
```
Comment (SaschaMann) [inline .github/labeler.yml:20]: These are based on the v3 labeler.yml file and may be incomplete

**Comment by SleeplessByte** [comment]
Is this substantially different than #1281 ?

**Comment by SaschaMann** [comment]
No, I didn't know about that PR (didn't look for any). But labeler isn't enabled, so whatever labeler.yml we go with, we should add the workflow file.

File: .github/labeler.yml [hunk 2]
```diff
@@ -10,3 +16,116 @@ type/mentor-bios:
...
+
+track/bash:
+- **/bash/**/*
+
+track/c:
+- **/c/**/*
+
```
Comment (angelikatyborska) [inline .github/labeler.yml:27]:
```suggestion

track/ceylon:
- **/ceylon/**/*

track/cfml:
- **/cfml/**/*

track/clojure:
- **/clojure/**/*

track/coffeescript:
- **/coffeescript/**/*

```

File: .github/labeler.yml [hunk 3]
```diff
@@ -10,3 +16,116 @@ type/mentor-bios:
...
+track/common-lisp:
+- **/common-lisp/**/*
+
```
Comment (angelikatyborska) [inline .github/labeler.yml:30]:
```suggestion

track/coq:
- **/coq/**/*

```

File: .github/labeler.yml [hunk 4]
```diff
@@ -10,3 +16,116 @@ type/mentor-bios:
...
+track/cpp:
+- **/cpp/**/*
+
```
Comment (angelikatyborska) [inline .github/labeler.yml:33]:
```suggestion

track/crystal:
- **/crystal/**/*

```

File: .github/labeler.yml [hunk 5]
```diff
@@ -10,3 +16,116 @@ type/mentor-bios:
...
+track/csharp:
+- **/csharp/**/*
+
```
Comment (angelikatyborska) [inline .github/labeler.yml:36]:
```suggestion

track/d:
- **/d/**/*

```

File: .github/labeler.yml [hunk 6]
```diff
@@ -10,3 +16,116 @@ type/mentor-bios:
...
+track/emacs-lisp:
+- **/emacs-lisp/**/*
+
+track/erlang:
+- **/erlang/**/*
+
+track/factor:
+- **/factor/**/*
+
```
Comment (angelikatyborska) [inline .github/labeler.yml:57]:
```suggestion

track/fortran:
- **/fortran/**/*

```

File: .github/labeler.yml [hunk 7]
```diff
@@ -10,3 +16,116 @@ type/mentor-bios:
...
+track/fsharp:
+- **/fsharp/**/*
+
```
Comment (angelikatyborska) [inline .github/labeler.yml:60]:
```suggestion

track/gleam:
- **/gleam/**/*

track/gnu-api:
- **/gnu-api/**/*

```

File: .github/labeler.yml [hunk 8]
```diff
@@ -10,3 +16,116 @@ type/mentor-bios:
...
+track/go:
+- **/go/**/*
+
```
Comment (angelikatyborska) [inline .github/labeler.yml:63]:
```suggestion

track/groovy:
- **/groovy/**/*

```

File: .github/labeler.yml [hunk 9]
```diff
@@ -10,3 +16,116 @@ type/mentor-bios:
...
+track/haskell:
+- **/haskell/**/*
+
```
Comment (angelikatyborska) [inline .github/labeler.yml:66]:
```suggestion

track/haxe:
- **/haxe/**/*

track/idris:
- **/idris/**/*

```

File: .github/labeler.yml [hunk 10]
```diff
@@ -10,3 +16,116 @@ type/mentor-bios:
...
+track/javascript:
+- **/javascript/**/*
+
+track/julia:
+- **/julia/**/*
+
+track/kotlin:
+- **/kotlin/**/*
+
```
Comment (angelikatyborska) [inline .github/labeler.yml:81]:
```suggestion

track/lfe:
- **/lfe/**/*

track/lua:
- **/lua/**/*

track/mips:
- **/mips/**/*

```

File: .github/labeler.yml [hunk 11]
```diff
@@ -10,3 +16,116 @@ type/mentor-bios:
...
+track/nim:
+- **/nim/**/*
+
```
Comment (angelikatyborska) [inline .github/labeler.yml:84]:
```suggestion

track/objective-c:
- **/objective-c/**/*

```

File: .github/labeler.yml [hunk 12]
```diff
@@ -10,3 +16,116 @@ type/mentor-bios:
...
+track/ocaml:
+- **/ocaml/**/*
+
+track/perl5:
+- **/perl5/**/*
+
+track/pharo-smalltalk:
+- **/pharo-smalltalk/**/*
+
```
Comment (angelikatyborska) [inline .github/labeler.yml:93]:
```suggestion

track/php:
- **/php/**/*

track/plsql:
- **/plsql/**/*

track/pony:
- **/pony/**/*

track/powershell:
- **/powershell/**/*

track/prolog:
- **/prolog/**/*

```

File: .github/labeler.yml [hunk 13]
```diff
@@ -10,3 +16,116 @@ type/mentor-bios:
...
+- **/python/**/*
+
+track/r:
+- **/r/**/*
+
+track/raku:
+- **/raku/**/*
+
+track/racket:
+- **/racket/**/*
```
Comment (angelikatyborska) [inline .github/labeler.yml:107]: Alphabetically:
```suggestion
track/racket:
- **/racket/**/*

track/raku:
- **/raku/**/*
```

File: .github/labeler.yml [hunk 14]
```diff
@@ -10,3 +16,116 @@ type/mentor-bios:
...
+
+track/rust:
+- **/rust/**/*
+
+track/scala:
+- **/scala/**/*
+
+track/scheme:
+- **/scheme/**/*
+
```
Comment (angelikatyborska) [inline .github/labeler.yml:123]:
```suggestion

track/sml:
- **/sml/**/*

```

File: .github/labeler.yml [hunk 15]
```diff
@@ -10,3 +16,116 @@ type/mentor-bios:
...
+track/swift:
+- **/swift/**/*
+
```
Comment (angelikatyborska) [inline .github/labeler.yml:126]:
```suggestion

track/tcl:
- **/tcl/**/*

```

File: .github/labeler.yml [hunk 16]
```diff
@@ -10,3 +16,116 @@ type/mentor-bios:
...
+track/typescript:
+- **/typescript/**/*
+
```
Comment (angelikatyborska) [inline .github/labeler.yml:129]:
```suggestion

track/vbnet:
- **/vbnet/**/*

track/vimscript:
- **/vimscript/**/*

```

**Review by angelikatyborska — COMMENTED** [review]
I went through `/mentors` and added suggestions to add all languages that have a directory there. I wasn't sure if labels in this repo are automatically synced, doesn't look like it to me, so I went through them and created labels that were missing, so that every label listed in `.github/labeler.yml` exists.

I would love to see this feature get merged soon! 🚀

**Comment by SaschaMann** [comment]
> I wasn't sure if labels in this repo are automatically synced, doesn't look like it to me, so I went through them and created labels that were missing

To save some work in the future: labels are automatically created if they're added through the API, so they'd be created over time.

Thanks for adding them all and reviving the PR!

**Review by angelikatyborska — APPROVED** [review]

**Comment by angelikatyborska** [comment]
Something went wrong: https://github.com/exercism/website-copy/pull/1945

**Comment by angelikatyborska** [comment]
I had no idea about this, but looks like `*` and `&` have a special meaning in YAML: https://medium.com/@kinghuang/docker-compose-anchors-aliases-extensions-a1e4105d70bd

**Pull request merged**